docs/usage/tracing.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[ecs-tracing-usage]]
+==== Tracing Usage
+
+A trace describes the individual operations, and their causality, that ensue as a result of a single logical 
+operation. A distributed trace comprises operations across multiple distributed components, crossing process, 
+network, and security boundaries.
+
+For example, a distributed trace may be initiated when someone presses a button to start an action on a website - in this example, 
+the trace will represent calls made between the downstream services that handled the chain of requests initiated by this 
+button being pressed.
+
+A trace is made up of transactions and spans, where each transaction/span is one operation, activity, or part of the overall
+trace. A transaction is a special kind of span that is the highest level of work measured within a service. In Elastic APM, 
+transactions have additional attributes associated with them 
+(for more detail see https://www.elastic.co/guide/en/apm/get-started/current/transactions.html).
```

Comment:
We use attributes for internal links. This ensures links are always correct -- even if pages move.

```suggestion
Learn more about transactions in {apm-overview-ref-v}/transactions.html[APM Transactions].
```